pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -685,11 +703,11 @@ var _ = Describe("Converter", func() {
 		})
 
 		DescribeTable("should use virtio-transitional models if requested", func(arch string) {
+			c.Architecture = NewArchConverter(arch)
```

Comment:
Add check for s390x as described in TODO in L718?

Reply:
As per comment above, would do in a follow up